packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -181,7 +181,7 @@ function BlockListBlock( {
 			} = select( blockEditorStore );
 			const currentBlockType = getBlockType( name || 'core/missing' );
 			const blockOrder = getBlockIndex( clientId );
-			const innerBlockSelected = hasSelectedInnerBlock( clientId );
+			const innerBlockSelected = hasSelectedInnerBlock( clientId, true );
```

Comment:
Additionally, it may have no impact, but we invoke `hasSelectedInnerBlock( clientId, true )` again further down in this selector for `isDraggingEnabled`. Is it beneficial to reuse the result of the first invocation?

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/50108/commits/2146fed259367ea0f706f6123faa2b668a012032